### ISSUE TITLE
Correction in naming the organisation 

### ISSUE THREAD

**Issue by AjeyAshok** [nusCS2113-AY1920S1/forum#56]
I made a small error of not adding the letter 'T' to the module code while creating the organisation and therefore the bot was unable to recognize our organisation repo. Is it possible to request for a re-run of the checking as we managed to send all the necessary submissions on time?

**Comment by okkhoy** [comment]
No. Sorry! As mentioned in the lecture a couple of weeks ago. We have to work within limits imposed by GitHub. Hence, it is impossible to rerun the scripts for individual errors. Please rename it appropriately, and we will pick up the updates next week.
